### DISCUSSION TITLE
I deleted "ServerTypings" folder and  transforming templates stopped working

### DISCUSSION THREAD

**Discussion by ghost** [serenity-is/Serenity#5981] [Help] [answered]
Hey, I modified XYZRequest.cs file and when I built my project, its typescript version was not updated so I deleted the entire ServerTypings folder which renderd my entire TS code useless because none of my .cs files get transformed into theirs .ts versions.

command line gives me this warning:
`The command "dotnet sergen servertypings" exited with code -1073741819.`

However I cannot confirm that this warning appeared after me deleting ServerTypings

Any ideas? Thanks

**Comment by VictorTomaili** [comment, marked as answer]
create a folder named `ServerTypings` and rebuild your application several times probably can fix this problem.

**Reply by ghost** [reply]
Thank you!
